nextcloudappstore/core/models.py

```diff
@@ -383,6 +381,18 @@ def is_outdated(self):
             "webhook_listeners",
             "workflowengine",
         ]:
+            return True
+        return False
+
+    def is_outdated(self):
+        """Checks if an app has been released in last 3 recent platform versions
+
+        :return: True if not compatible, otherwise false
+        """
+
+        if self.is_integration:
+            return False
+        if self.is_server_bundled:
```

Comment:
Maybe
```suggestion
        if self.is_server_bundled():
```

as per pre-commit?
But not sure if that also works in the template